src/types/severa/workDays/workDays.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Interface for WorkDaysModel
+ */
+interface WorkDaysModel {
+  userGuid: string;
```

Comment:
this should be names severaUserId
Please, also heck other occurencies of IDs from severa